mappings/net/minecraft/block/MapColor.mapping

```diff
@@ -0,0 +1,94 @@
+CLASS net/minecraft/unmapped/C_mlrprqox net/minecraft/block/MapColor
+	FIELD f_aaztdeah LAPIS Lnet/minecraft/unmapped/C_mlrprqox;
+	FIELD f_abkjjtly NONE Lnet/minecraft/unmapped/C_mlrprqox;
+	FIELD f_ahkfeptz TERRACOTTA_GRAY Lnet/minecraft/unmapped/C_mlrprqox;
```

Comment:
Hm, why not "COLOR_TERRACOTTA"?

Reply:
good point, that's the more natural form